blocks/library/button/index.js

```diff
@@ -74,7 +74,7 @@ registerBlock( 'core/button', {
 			<span className="blocks-button" title={ title }>
 				<Editable
 					tagName="span"
-					placeholder={ wp.i18n.__( 'Write some text…' ) }
+					placeholder={ wp.i18n.__( 'Write label…' ) }
```

Comment:
Yeah, that's what I mean with my comment in the other PR. Will have a look, atm I see no easy fix. I tried without the position absolute, and also with :after, but that doesn't really work well. This is a sting issue really. :)